[MC-284] Added JSON object containing the additional fields and values to create / update incident (#815)
rapid7/insightconnect-plugins: servicenow/help.md

# Version History

> * 5.0.1 - Add JSON object containing the additional fields and values to create / update incident

Reviewer: ```suggestion
* 5.0.1 - Add new Additional Fields input for Create Incident and Update Incident actions
```
